samples/csharp_dotnetcore/70.skills-simple-bot-to-bot/README.md

```diff
@@ -0,0 +1,27 @@
+# SimpleBotToBot (**DRAFT**)
+
+Bot Framework v4 Skills Echo sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to use skills from a rootbot.
```

Comment:
Or maybe it should be, "...from a host (root) bot." Or, some other wording altogether.
```suggestion
This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to use skills from a root (host) bot.
```

Reply:
Addressed